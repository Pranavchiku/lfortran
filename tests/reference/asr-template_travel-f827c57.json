{
    "basename": "asr-template_travel-f827c57",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/template_travel.f90",
    "infile_hash": "7455a7b02d2507204b88f4d57905bebed0b4c3906a3b468f02a1ee44",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-template_travel-f827c57.stdout",
    "stdout_hash": "f029519dd6bbb61c92a51550e1cf757cf846b3ffe2723a56a502a147",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}