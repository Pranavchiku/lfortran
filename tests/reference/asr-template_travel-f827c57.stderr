warning: Implied save feature
 --> tests/../integration_tests/template_travel.f90:5:5
  |
5 |     public :: add_real, slash_real
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/template_travel.f90:5:5
  |
5 |     public :: add_real, slash_real
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:10:9
   |
10 |         real, intent(in) :: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:10:9
   |
10 |         real, intent(in) :: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:11:9
   |
11 |         real :: total
   |         ^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:16:9
   |
16 |         real, intent(in) :: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:16:9
   |
16 |         real, intent(in) :: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:17:9
   |
17 |         real :: total
   |         ^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:28:5
   |
28 |     public :: travel_tmpl
   |     ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:36:13
   |
36 |             type(D), intent(in) :: l, R
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:36:13
   |
36 |             type(D), intent(in) :: l, R
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:37:13
   |
37 |             type(D) :: total
   |             ^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:41:13
   |
41 |             type(T), intent(in) :: l, R
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:41:13
   |
41 |             type(T), intent(in) :: l, R
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:42:13
   |
42 |             type(T) :: total
   |             ^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:46:13
   |
46 |             type(D), intent(in) :: n
   |             ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:47:13
   |
47 |             type(T), intent(in) :: d
   |             ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:48:13
   |
48 |             type(S) :: quotient
   |             ^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:52:13
   |
52 |             type(D), intent(in) :: n
   |             ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:53:13
   |
53 |             type(S), intent(in) :: d
   |             ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:54:13
   |
54 |             type(T) :: quotient
   |             ^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:61:9
   |
61 |         public :: avg_S_from_T
   |         ^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:64:13
   |
64 |             type(D), intent(in) :: d1, d2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:64:13
   |
64 |             type(D), intent(in) :: d1, d2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:65:13
   |
65 |             type(T), intent(in) :: t1, t2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:65:13
   |
65 |             type(T), intent(in) :: t1, t2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:66:13
   |
66 |             type(S) :: avg
   |             ^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:71:13
   |
71 |             type(D), intent(in) :: d1, d2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:71:13
   |
71 |             type(D), intent(in) :: d1, d2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:72:13
   |
72 |             type(S), intent(in) :: s1, s2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:72:13
   |
72 |             type(S), intent(in) :: s1, s2
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:73:13
   |
73 |             type(S) :: avg
   |             ^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:93:9
   |
93 |         real :: s1, s2
   |         ^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/template_travel.f90:93:9
   |
93 |         real :: s1, s2
   |         ^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
